create.js

```diff
@@ -0,0 +1,10 @@
+const employee = {
+  name: 'ANE01',
+  email: 'Anna@example.com',
+  phone: '01123456789',
+  isMale: true,
+};
+
+const name = employee.name;
```

Comment:
Hey i would like u to minimize you code a little, so if you do destructuring on the Variable name like this:
Const {name} = employee i would be very pleaseed.